subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/configuration/GradleInstrumentationPluginConfiguration.kt

```diff
@@ -34,6 +34,9 @@ object InstrumentationPluginConfiguration {
 
         var unitToChannelMap: Map<String, String> = emptyMap()
 
+        var applicationProguardMapping: String? = null
```

Comment:
Is it mapping or path to mapping? I consider the type of this `File?`

Reply:
I did it like other properties around.

Reply:
Replace `String` with `File`